Blockzilla/URLBar.swift

```diff
@@ -136,8 +137,12 @@ class URLBar: UIView {
 
     convenience init() {
         self.init(frame: CGRect.zero)
+        
+        let urlBarSingleTap = UITapGestureRecognizer(target: self, action: #selector(didSingleTapUrlBar(sender:)))
+        urlBarSingleTap.numberOfTapsRequired = 1
```

Comment:
the default value for `numberOfTapsRequired` is 1

Reply:
Also below for the other `UITapGestureRecognizer`.